model-optimizer/extensions/front/onnx/non_max_suppression_normalize.py

```diff
@@ -30,10 +30,10 @@ class NonMaxSuppressionNormalize(FrontReplacementSubgraph):
 
     def find_and_replace_pattern(self, graph: Graph):
         for nms in graph.get_op_nodes(op='NonMaxSuppression'):
-            # make inputs 2 to 4 to have shape [] instead of [1] (convert 1D to 0D)
```

Comment:
Update the comment in the transformation description.

Reply:
Done.